pkg/piecestore/psserver/server_test.go

```diff
@@ -649,95 +591,45 @@ func newTestServerStruct(t *testing.T, ids []storj.NodeID) (*Server, func()) {
 		totalBwAllocated: math.MaxInt64,
 		whitelist:        ids,
 	}
-	return server, func() {
-		if serr := server.Stop(context.TODO()); serr != nil {
-			t.Fatal(serr)
-		}
-		// TODO:fix this error check
-		_ = os.RemoveAll(tmp)
-		// if err := os.RemoveAll(tmp); err != nil {
-		// 	t.Fatal(err)
-		// }
+	//init ps server grpc
+	listener, err := net.Listen("tcp", "127.0.0.1:0")
+	assert.NoError(t, err)
```

Comment:
not a big deal but you probably want require.NoError for this type of code style. assert.NoError marks the test as failed but lets the test keep going. require.NoError halts the test.

Reply:
i do see that this affects most of this test suite

Reply:
Ah!  I didn't know about that one